manifests/install_java.pp

```diff
@@ -28,26 +28,19 @@
   $jdk_version        = '8',
   $jdk_version_update = '221',
   $jdk_version_build  = '',
-  $jdk_format         = 'rpm',
-) {
+)  {
 
-  class { 'oracle_java':
-    download_url   => $jdk_base_url,
-    filename       => $jdk_filename,
-    version        => "${jdk_version}u${jdk_version_update}",
-    build          => $jdk_version_build,
-    type           => 'jdk',
-    format         => $jdk_format,
-    check_checksum => false,
-  # Need to set alternative for java here due to oracle_java module's add alternative feature is broken in version 2.9.4
-  } -> exec { "alternatives --set  java /usr/java/jdk1.${jdk_version}.0_${jdk_version_update}-amd64/jre/bin/java":
-    path => [ '/bin', '/sbin', '/usr/bin', '/usr/sbin' ],
-  }
+    java::download { $jdk_version :
+      ensure  => 'present',
+      java_se => 'jdk',
+      url     => "${jdk_base_url}/${jdk_filename}",
+    }
 
   file { '/etc/ld.so.conf.d/99-libjvm.conf':
     ensure  => present,
-    content => "/usr/java/latest/jre/lib/amd64/server\n",
+    content => "/usr/java/latest/lib/server\n",
```

Comment:
Is this path unique to a JDK version ? if yes please set a variable with the correct parameter depending on the JDK version Thanks

Reply:
Our new module create this path which linked to jdk_install_path, so it work fine with both jdk11 and jdk8